api/client/cli.go

```diff
@@ -105,15 +105,21 @@ func NewDockerCli(in io.ReadCloser, out, err io.Writer, clientFlags *cli.ClientF
 
 		switch len(hosts) {
 		case 0:
-			hosts = []string{os.Getenv("DOCKER_HOST")}
+			defaultHost := os.Getenv("DOCKER_HOST")
```

Comment:
yes, you are right, `defaultHost` is useless here, `hosts = []string{os.Getenv("DOCKER_HOST")}` has the same result